src/themes/dark/oled.ts

```diff
@@ -0,0 +1,19 @@
+import colors from 'vuetify/util/colors'
+import { getVariables } from '../global'
+
+export default {
+  id: 'dark-oled',
+  theme: {
+    dark: true,
+    colors: {
+      primary: '#35495E',
+      secondary: '#415c75',
+      navbar: '#273845',
+      download: '#5BB974',
+      background: '#121212',
+      selected: colors.grey.darken1,
+      red: colors.red.accent3,
+      ...getVariables(true)
+    }
```

Comment:
This object can be imported from the other theme to prevent duplication. You can then override the `torrent-ul_stalled` color with a more color suitable for OLED